.github/workflows/verify.yml

```diff
@@ -14,9 +14,6 @@ jobs:
     steps:
     - name: Verifier action
       id: verifier
-      uses: kubernetes-sigs/kubebuilder-release-tools@v0.2.0
-      # Don't run this step on PRs created by the cherry-pick bot,
-      # as the PR description is too short, but otherwise the PRs are fine.
```

Comment:
The descriptiveness check was dropped: https://github.com/kubernetes-sigs/kubebuilder-release-tools/pull/49